otkerneldesign/KernelHerding.py

```diff
@@ -33,6 +33,10 @@ class KernelHerding:
         If not specified, then *distribution* and *candidate_set_size* must be in order to generate it automatically.
     initial_design : 2-d list of float
         Sample of points that must be included in the design. Empty by default.
+    is_greedy : Boolean 

```

Comment:
I would advise dropping this option, as your experiments show it does not change much.